xprof/Makefile.am

```diff
@@ -20,8 +20,19 @@ xprof_utils.hpp: $(top_srcdir)/utils/xprof_utils.hpp
 %.pb.h %.pb.cc: %.proto
 	$(PROTOC) -I=$(srcdir) --cpp_out=$(builddir) $<
 
-%/metababel/metababel.h %/metababel/btx_component.h %/metababel/btx_component.c %/metababel/btx_upstream.h %/metababel/btx_upstream.c %/btx_main.c: %_upstream_model.yaml
-	$(METABABEL) -u $< -t SINK -o $* -p $* -c $*
+btx_tally_upstream_model.yaml btx_timeline_upstream_model.yaml: btx_interval_model.yaml
```

Comment:
Same problem, we cannot have 2 output in make. It will break parallel build.

Reply:
Just to be sure I am getting the right message.

`btx_tally_upstream_model.yaml btx_timeline_upstream_model.yaml: btx_interval_model.yaml
`
is simular to 

`%.pb.h %.pb.cc: %.proto
`
So both will breack the parallel build ?

Reply:
What @Kerilk  is saying is that the `%.` is more robust to parallel build.